### PR TITLE
updating changelog for version 2.3.0

### PR DESCRIPTION
**Pull request by danifr**

### CODE REVIEW

**Comment by jyaworski** [comment]
@danifr LGTM. You should hop into #voxpupuli on Freenode and ping myself or @bastelfreak.

**Comment by bastelfreak** [comment]
looks good. I would like to merge the modulesync PR before we do the new release.

**Comment by bastelfreak** [comment]
we just merged the modulesync in https://github.com/voxpupuli/puppet-rundeck/pull/233

**Comment by bastelfreak** [comment]
can you merge master into your branch to fix the travis issues?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
+#2016-05-26 - Release 2.3.0
```
Comment (bastelfreak) [inline CHANGELOG.md]: could you add a whitespace after the # please?

**Pull request merged**